nom-sql/src/show.rs

```diff
@@ -19,7 +19,7 @@ pub enum ShowStatement {
     Events,
     Tables(Tables),
     CachedQueries(Option<QueryID>),
-    ProxiedQueries(Option<QueryID>),
+    ProxiedQueries(Option<QueryID>, Option<bool>),
```

Comment:
a bare `Option<bool>` is a little opaque imo - maybe it's time to convert this enum variant into a struct?

Reply:
also it doesn't seem like we're doing anything in particular with the fact that this is an `Option` - could it not just be a bare `bool`?

Reply:
You are right, `bool` here is not optional. It should be always true or false.
I changed it to bare `bool` and encapsulated the options into a struct.

Reply:
this works, but also see https://github.com/readysettech/readyset/pull/429#discussion_r1305713889